db/seeds.rb

```diff
@@ -63,6 +63,15 @@
       "info" => { "nickname" => user.username }
     },
   )
+
+  # Add some users to organizations.
+  next unless rand(4) == 1
+
+  OrganizationMembership.create!(
+    user_id: user.id,
+    organization_id: Organization.all.sample.id,
+    type_of_user: %i[member admin].sample,
+  )
```

Comment:
there's a tiny chance that orgs will have only admins or only members. What if instead of using random, we just make it a little bit more deterministic?

After creating the users you can:

- select N orgs
- for each org you create 1 admin and N members

Reply:
Would love your opinion on the new approach.